main.go

```diff
@@ -23,18 +23,22 @@ func main() {
 		command = os.Args[1]
 	}
 
+	if !config.Cfg.RunMigrations && command != "migrate" {
```

Comment:
i'd prefer this be wet and still called inside each switch. someone adding a switch in the future may accidentally wreck a db not knowing this auto runs

Reply:
after thinking about it some more, i feel like this might be a non-issue or at least quite a rare occurrence. for it to be a problem, you'd have to:
1. Be connected to an existing db, like prod/stage
2. Have `runMigrations=true`
3. Have a new command you're adding
4. Have a bad new migration that does something you don't want to do on those dbs

I think it's probably easier to reason around having _any_ command will run migrations if `runMigrations=true` vs all the special casing

Reply:
fair enough

Reply:
this is equivalent to what it was before, besides elasticsearch. It makes a lot of sense for the dev flow, but I agree is a bit overkill for deploys. We could pick a single command to champion migrations but I think it's alright for now